### PR TITLE
Ajw update models

### PR DESCRIPTION
**Pull request by awieckert**
# Name - PR
AJW-UpdateModels
##### Provide clarity to any potentially complex code:
Created the Boilerplate models in C# for our data tables
##### Explain reasons behind organizational or architectual decisions you made:
Created the bare minimum properties on the Models. Made sure all Model have the same properties as the data tables
## Steps to Test
Pull down and verify that all the models are visible and correct base properties are there.

##### 3rd party libraries that need to be installed.
None
##### Link to Feature Ticket
#23

### CODE REVIEW

**Comment by elliot-huck** [comment]
- [ ] Change model class name 'Employees' and 'EmployeeComputers' to singular
- [ ] Convert float types to double
- [ ] Make ReturnDate on 'EmployeeComputer' and DecomissionDate on 'Computer' optionally NULL

File: Models/Training.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Training
```
Comment (stevebrownlee) [inline Models/Training.cs]: This type should be `TrainingProgram`

File: Models/Training.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
```
Comment (stevebrownlee) [inline Models/Training.cs]: Do you know what this annontation is doing?  Why is this here?

File: Models/EmployeeComputer.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+        [Key]
+        public int Id { get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime AssignmentDate { get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
```
Comment (stevebrownlee) [inline Models/EmployeeComputer.cs]: Why is this annotation here?

File: Models/Employee.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    {
+
+        [Key]
+        public int Id { get; set; }
+
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
```
Comment (stevebrownlee) [inline Models/Employee.cs]: Why is this annotation here?

File: Models/Computer.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        public int Id { get; set; }
+
+        public string Model { get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime PurchaseDate{ get; set; }
+
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
```
Comment (stevebrownlee) [inline Models/Computer.cs]: Why is this annotation here?

File: Models/Training.cs (hunk 1 above)
Comment (awieckert) [reply Models/Training.cs]: Spoke with Steve on the naming convention of Training. We identified that their is a table in our DB by the name of "Trainings" and are okay to proceed with the Training model.

**Comment by stevebrownlee** [comment]
Merge it! :shipit:
